packages/dicom-codec/src/codecs/index.js

```diff
@@ -96,7 +96,8 @@ function adaptImageInfo(imageInfo) {
     componentCount: samplesPerPixel,
     componentsPerPixel: samplesPerPixel,
     signed,
-    isSigned: signed
+    isSigned: signed,
+    isUsingColorTransform: samplesPerPixel>1,
```

Comment:
The isUsingColorTransform was mising, cause a missing variable exception to be thrown.